ci/requirements/environment.yml

```
@@ -7,23 +7,23 @@ dependencies:
   - boto3
   - bottleneck
   - cartopy
-  - cdms2
+  # - cdms2  # TODO: Waiting on python 3.11 support.
   - cfgrib
   - cftime
   - dask-core
   - distributed
   - flox
   - fsspec!=2021.7.0
-  - h5netcdf
-  - h5py
-  - hdf5
+  # - h5netcdf  # TODO: Waiting on python 3.11 support.
+  # - h5py  # TODO: Waiting on python 3.11 support.
+  # - hdf5  # TODO: Waiting on python 3.11 support.
   - hypothesis
   - iris
   - lxml  # Optional dep of pydap
   - matplotlib-base
   - nc-time-axis
-  - netcdf4
-  - numba
+  # - netcdf4  # TODO: Waiting on python 3.11 support.
```

same here: there *should* be wheels for py311 on `conda-forge`
```suggestion
  - netcdf4
```